driver-core/pom.xml

```diff
@@ -334,6 +334,13 @@
               <classpathDependencyExcludes>
                 <classpathDependencyExcludes>io.netty:netty-transport-native-epoll</classpathDependencyExcludes>
               </classpathDependencyExcludes>
+              <properties>
```

Comment:
Not so great that this needs to be repeated, we could follow the same pattern I recently introduced for the `isolated` profile where the profile is declared in the root pom and the child modules only override the profile where needed.  Can fix that now or in a later PR.